druid/src/main/java/org/apache/calcite/adapter/druid/DruidQuery.java

```diff
@@ -611,6 +627,12 @@ protected QuerySpec getQuery(RelDataType rowType, RexNode filter, List<RexNode>
     try {
       final JsonGenerator generator = factory.createGenerator(sw);
 
+      if (aggregations.isEmpty()) {
```

Comment:
Why do we need to move this block here? Timeseries? Could you update the comment?